client/src/client-only-routes/show-profile-or-four-oh-four.tsx

```diff
@@ -79,11 +77,9 @@ function ShowProfileOrFourOhFour({
   }
 
   return isEmpty(requestedUser) ? (
-    showLoading ? (
-      <Loader fullScreen={true} />
-    ) : (
+    <Suspense fallback={<Loader fullScreen={true} />}>
       <FourOhFour />
-    )
+    </Suspense>
   ) : (
     <Profile isSessionUser={isSessionUser} user={requestedUser} />
```

Comment:
Is the expected change this? : 
`    <Suspense fallback={<Loader fullScreen={true} />}>
      <Profile isSessionUser={isSessionUser} user={requestedUser} />
    </Suspense>
`